packages/bpk-component-split-input/src/BpkSplitInput-test.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import BpkSplitInput from './BpkSplitInput';
+
+const defaultProps = {
+  name: 'otpInput',
+  id: 'otpInput',
+  label: 'otp input',
+  onChange: () => {},
+  onSubmit: () => {},
+};
+
+describe('BpkSplitInput', () => {
+  it('should render without crashing', () => {
+    const wrapper = render(<BpkSplitInput {...defaultProps} />);
```

Comment:
The render method returns an object that has an `asFragment` prop. You can use that to compare only the tree snapshots (without all the other props) as such
```
    const { asFragment } = render(<BpkSplitInput {...defaultProps} />);
    expect(asFragment()). toMatchSnapshot();
```